vtex/billing-app
# Fix/install

#### What is the purpose of this pull request?
Allow accounts without Billing to install free apps

#### What problem is this solving?
Whitelabel and other accounts that do not have a proper Billing module fail to install apps because a request to Billing before the verification of ownership or publicity of an app would always return a 500 Internal Server Error. This change forces the billing app to verify existing Billling contracts only after it is certain that the app is **not**: property of the account where it is being installed, public, or free

#### How should this be manually tested?
1. Clone/pull this repo/branch and link in a workspace at account `gc_fdt5282`
2. Try this command
`vtex install vtex.pages-editor --verbose`
3. The installation of this app should be successfull

NOTE TO TOOLBELT:
This PR creates another status string: `public_app`. The other ones remain unchanged

#### Types of changes
